env/production/cwagent-configmap.yaml

```diff
@@ -0,0 +1,71 @@
+kind: ConfigMap
+metadata:
+  name: prometheus-cwagentconfig
+  namespace: amazon-cloudwatch
+apiVersion: v1
+data:
+  # cwagent json config
+  cwagentconfig.json: |
+    {
+      "agent": {
+        "region": "ca-central-1",
+        "debug": false
+      },
+      "logs": {
+        "metrics_collected": {
+          "prometheus": {
+            "cluster_name": "notification-canada-ca-production-eks-cluster",
+            "log_group_name": "/aws/containerinsights/notification-canada-ca-production-eks-cluster/prometheus",
```

Comment:
I wish we didn't have to hardcode these values but I can't think of a way to get the value dynamically either.

Reply:
I think you're right about the only option being `envsubst`.  Appreciate you digging into it more though.